docs/UserGuide.md

```diff
@@ -156,9 +155,12 @@ Format: `add id/EMPLOYEE_ID n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]… [
 
 - Name must be: [*Alphanumeric*](#alphanumeric), spaces allowed
 - Phone Number must be: [*Numeric*](#numeric), no spaces, at least 3 digits long
-- Email must be: A valid email address
+- Email must be: of the format `local-part@domain` and follow these constraints:
+  - The local-part should only contain alphanumeric characters and the special characters +_.-
+  - The local-part may not start or end with special characters.
+  - This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods (e.g. `u.nus.edu`).
 - Address must be: Any characters are valid
-- Skills and tags must be: [*Alphanumeric*](#alphanumeric), no spaces
+- Skills and tags must be: [*Alphanumeric*](#alphanumeric), no spaces, within 50 characters
```

Comment:
Perhaps it would be more accurate to say "50 characters or less"? Since "within 50 characters" seems to imply that skills/tags with exactly 50 characters will not be allowed?